cmd/podman/cp.go

```diff
@@ -273,26 +292,26 @@ func parsePath(runtime *libpod.Runtime, path string) (*libpod.Container, string)
 }
 
 func getPathInfo(path string) (string, os.FileInfo, error) {
-	path, err := filepath.EvalSymlinks(path)
+	cleanPath, err := filepath.EvalSymlinks(path)
 	if err != nil {
-		return "", nil, errors.Wrapf(err, "error evaluating symlinks %q", path)
+		return "", nil, errors.Wrapf(err, "error evaluating symlinks %s", path)
 	}
-	srcfi, err := os.Stat(path)
+	srcfi, err := os.Stat(cleanPath)
 	if err != nil {
-		return "", nil, errors.Wrapf(err, "error reading path %q", path)
+		return "", nil, errors.Wrapf(err, "error reading path %q", cleanPath)
 	}
-	return path, srcfi, nil
+	return cleanPath, srcfi, nil
 }
 
-func copy(src, destPath, dest string, idMappingOpts storage.IDMappingOptions, chownOpts *idtools.IDPair, extract, isFromHostToCtr bool) error {
-	srcPath, err := filepath.EvalSymlinks(src)
-	if err != nil {
-		return errors.Wrapf(err, "error evaluating symlinks %q", srcPath)
-	}
-
-	srcPath, srcfi, err := getPathInfo(srcPath)
-	if err != nil {
-		return err
+func copy(src, destPath, dest string, idMappingOpts storage.IDMappingOptions, chownOpts *idtools.IDPair, extract, isFromHostToCtr bool) (err error) {
+	// Don't symlink process /dev/stdin
+	srcPath := os.Stdin.Name()
+	srcfi, _ := os.Stat(srcPath)
```

Comment:
This is an extra system call on `/dev/stdin` we do even in cases when `src` is not `/dev/stdin`; it would be nice to avoid that. (If this code needs to exist at all.)